resources/user-agents/apps/microsoft/bing-search-app-android-4.json

```diff
@@ -0,0 +1,53 @@
+{
+    "division": "Bing Search App #MAJORVER#.#MINORVER# for Android",
+    "versions": ["4.3", "4.2"],
+    "sortIndex": 484,
+    "lite": false,
+    "standard": true,
+    "userAgents": [
+        {
+            "userAgent": "Bing Search App #MAJORVER#.#MINORVER# for Android",
+            "engine": "Blink",
+            "device": "general Mobile Phone (Touch)",
+            "platform": "Android",
+            "properties": {
+                "Parent": "DefaultProperties",
+                "Comment": "Bing Search App #MAJORVER#.#MINORVER# for Android",
+                "Browser": "Bing Search App",
+                "Browser_Type": "Application",
+                "Browser_Maker": "Microsoft Corporation",
+                "Version": "#MAJORVER#.#MINORVER#",
+                "Crawler": "false"
+            },
+            "children": [
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "devices": {
+                        "VK810 4G": "LG VK810"
+                    },
+                    "platforms": [
+                        "Android_5_0"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "devices": {
+                        "HTC D310w": "HTC Desire 310"
+                    },
+                    "platforms": [
+                        "Android_4_2"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "platforms": [
+                        "Android_4_2",
```

Comment:
Since there's a 5.0 pattern above (with a device) shouldn't a device-less `Android_5_0` be added here as well?

Reply:
changed